extensions/hibernate-orm/runtime/src/main/java/io/quarkus/hibernate/orm/runtime/DefaultEntityManagerFactoryProducer.java

```diff
@@ -1,11 +1,9 @@
 package io.quarkus.hibernate.orm.runtime;
 
 import javax.enterprise.inject.Produces;
-import javax.inject.Singleton;
 import javax.persistence.EntityManagerFactory;
 import javax.persistence.PersistenceUnit;
 
-@Singleton
```

Comment:
Why do you remove the scope here? Since the produced `EntityManagerFactory` is dependent a new instance of this producer will be created for each `EntityManagerFactory`  injection point...

Reply:
I could have sworn the producer also had a scope on it, oops.

Reply:
@mkouba are you good with the move of the annotation to the producer method? I would like to get this approved (I think all the changes make sense) and merged if CI passes.

Reply:
Looks good so +1 if CI passes ;-).